src/Cartography/Cartography.DynamicMap/MapControlBase.cs

```diff
@@ -493,8 +493,9 @@ private async Task TrySetViewPort(CancellationToken ct, MapViewPort viewPort)
 			}
 			catch (TaskCanceledException)
 			{
-				// This is a normal case, for instance if the user drags the map during the animation
-			}
+                // This is a normal case, for instance if the user drags the map during the animation
+                _isAnimating = false;
```

Comment:
since the issue is for android, what's the behaviour for ios

Reply:
Work on IOS side. It handle IsAnimated by himself before.